signer/standard_signer.go

```diff
@@ -199,7 +198,7 @@ func (s *StandardSigner) sign(template *x509.Certificate, profile *config.Signin
 		}
 	}
 	cert = pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: derBytes})
-	log.Infof("Signed certificate with serial number %s", serialNumber)
+	log.Infof("signed certificate with serial number %s", serialNumber)
```

Comment:
I think we should look at this in another branch. It requires some extra work that isn't orthogonal to this PR.